internal/confgen/document_parser.go

```diff
@@ -56,6 +56,13 @@ func (sp *documentParser) parseMessage(msg protoreflect.Message, node *book.Node
 				}
 			} else {
 				fieldNode = node.FindChild(field.opts.Name)
+				if fieldNode == nil && xproto.GetFieldDefaultValue(fd) != "" {
+					// if this field has a default value, use virtual node
+					fieldNode = &book.Node{
+						Name:  node.Name,
+						Value: node.Value,
+					}
+				}

```

Comment:
This is a greet feature, please add test cases to cover this.

Reply:
added